manifests/bridge-pod.yml

```diff
@@ -0,0 +1,11 @@
+apiVersion: v1
```

Comment:
We need to make the distinction in manifests/ between examples and actual deployment-related manifests.

Reply:
done